release notes/v0.51.0.md

```diff
@@ -27,19 +27,16 @@ _what, why, and what this means for the user_
 
 _Format as `<number> <present_verb> <object>. <credit>`_:
 
-- _`#999` Gives terminal output prettier printing. Thanks to `@person` for the help!_
-- `#pr` `<description>`
-- `#pr` `<description>`
+- [browser#1259](https://github.com/grafana/xk6-browser/pull/1259), [browser#1260](https://github.com/grafana/xk6-browser/pull/1260) adds errors to the traces that the browser module generates.
 
 ## Bug fixes
 
-_Format as `<number> <present_verb> <object>. <credit>`_:
-- _`#111` fixes race condition in runtime_
+- [browser#1261](https://github.com/grafana/xk6-browser/pull/1261) fixes dispose context canceled errors.
 
 ## Maintenance and internal improvements
 
-_Format as `<number> <present_verb> <object>. <credit>`_:
-- _`#2770` Refactors parts of the JS module._
+- [browser#1262](https://github.com/grafana/xk6-browser/pull/1262) fixes a flakey test.
```

Comment:
flaky?

Reply:
Yep, changed in 1248e4dd10e4ff14a557e789c55117e0ab808f51